layouts/shortcodes/image-width.html

```diff
@@ -1,4 +1,6 @@
 <!-- This shortcode adds an image with a maximum width specified in pixels and an alt text caption. Particularly useful for .svgs that can scale infinitely.
 -->
 
-<img src={{ .Get 0 }} width={{ .Get 1 }} alt={{ .Get 2 }}>
\ No newline at end of file
+<img src={{ .Get 0 }} width={{ .Get 1 }} alt={{ .Get 2 }}>
+</br>
+</br>

```

Comment:
Nit: It's just \<br\>, no leading `/`.
```suggestion
<br>
<br>
```